assets/src/styles/blocks/Accordion/AccordionStyle.scss

```diff
@@ -5,18 +5,25 @@
 
 .accordion-block {
   _-- {
-    font-family: var(--headings--font-family);
+    font-family: var(--font-family-primary);
   }
 
   &.is-style-light .accordion-content .accordion-headline {
-    --accordion-block-light-style-- {
-      border-color: $grey-20;
-      background: $white;
-      color: var(--body--color);
```

Comment:
I think this line should stay, right now the font color is white in this case and it's hard to read